src/app/landing-page/landing-page.component.html

```diff
@@ -1,348 +1,425 @@
 <script src="../../environments/environment.prod.ts"></script>
-<div class="w-100">
-  <div class="container d-sm-none">
-    <div class="row d-flex justify-content-center landing__ticker">
-      <div class="input-group w-auto">
-        <div class="input-group-prepend">
-          <label class="landing__nav-buy input-group-text"
-            >{{ globalVars.desoToUSDExchangeRateToDisplay }} USD</label
+
+<section>
+  <div
+    *ngIf="environment.node.name === 'bitclout'; else otherEnvironments"
```

Comment:
```suggestion
    *ngIf="environment.node.name === 'BitClout'; else otherEnvironments"
```
According to `environment.bitclout.ts`, the node name is `BitClout`